includes/wf_crm_admin_form.inc

```diff
@@ -321,6 +323,39 @@ class wf_crm_admin_form {
             $fieldset[$fid] = $this->addItem($fid, $field);
           }
         }
+
+        $isMultipleCustomGroup = FALSE;
+
+        if (substr($sid, 0, 2) == 'cg') {
+          $customGroupId = (int) substr($sid, 2);
+          $customGroup = $this->getCustomGroup($customGroupId);
+          if (!empty($customGroup['is_multiple']) && (int) $customGroup['is_multiple']) {
+            $isMultipleCustomGroup = TRUE;
+          }
+        }
+
+        // Add 'Create mode' select field to multiple custom group fieldset.
+        if ($isMultipleCustomGroup) {
```

Comment:
None of the above code defining `$isMultipleCustomGroup` is necessary. It would be sufficient to change this if statement to read `if (substr($sid, 0, 2) == 'cg' && wf_crm_aval($set, 'max_instances') > 1) {`